experimental/python/codegen/codegen/main.py

```diff
@@ -77,13 +77,17 @@ def _transitively_mark_deprecated_and_private(
 def _remove_deprecated_fields(
     schemas: dict[str, openapi.Schema],
 ) -> dict[str, openapi.Schema]:
+    """
+    Remove fields that were deprecated during Private Preview.
+    """
+
```

Comment:
Do you have examples of these?

They should rather be removed from the spec entirely.